<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_borderSize_width">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BoardSize (Width)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_borderSize_height">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BoardSize (Height)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_square_size">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Square Size(pix,millimeter)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pattern">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration Pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_frames">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Frames</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_detection">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Write detection Feautre points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_extr">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Write extrinstic parameters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_showund">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show undictorted images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_width">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_height">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_size">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>26</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_CHESS">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Chessboard</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_CIRCLE">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Circle Grid</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ASYMM">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Asymmetric grid</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton_lap">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Laptop</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_usb">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Usb</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_input">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_touch">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Images</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>281</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_chess">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_number">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_det">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_extr">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_undist">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
